website/content/api-docs/hcp-link.mdx

```diff
@@ -0,0 +1,186 @@
+[agent caching](/consul/api-docs/features/caching), and
+[required ACLs](/consul/api-docs/api-structure#authentication).
+
+| Blocking Queries | Consistency Modes     | Agent Caching | ACL Required                   |
+| ---------------- | --------------------- | ------------- | ------------------------------ |
+| `NO`             | `stale`, `consistent` | `none`        | `operator:write`, `acl:write`  |
+
+### JSON Request Body Schema
+
+- `data` `(object: <required>)`
+
+  - `resourceId` `(string: <required>)` - The ID of the HCP Consul Central cluster to link to. Has the format of:
+    `organization/<orgID>/project/<projectID>/hashicorp.consul.global-network-manager.cluster/<clusterName>`
+
+  - `clientId` `(string: <required>)` - The ID used to authenticate to HCP, which is returned as part of the HCP
+    Consul Central cluster creation. It can also be obtained by fetching the HCP Consul Central cluster secrets.
+
+  - `clientSecret` `(string: <required>)` - The secret used to authenticate to HCP, which is returned as part of the
+    HCP Consul Central cluster creation. It can also be obtained by fetching the HCP Consul Central cluster secrets.
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "resourceId": "organization/c0bf7aac-7690-4905-a8aa-889df1510314/project/6e82a47b-79af-4920-ad42-c0f74421ab52/hashicorp.consul.global-network-manager.cluster/my-cluster",
+    "clientId": "3jz6zk2tlr802htzwquczlxlsrohlpm5",
+    "clientSecret": "07ywcvaqmg1f8ko9eq4julep2tfglnv4o18rz3py9dik4ywox0ytoscycn39o4vs"
+  }
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl --request PUT \
+   --header "X-Consul-Token: 9cdjse6c-2dia-3720-81fe-5dae3k714a6e" \
+   --data @payload.json \
+   http://127.0.0.1:8500/api/hcp/v2/link/global
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "clientId": "3jz6zk2tlr802htzwquczlxlsrohlpm5",
+    "clientSecret": "07ywcvaqmg1f8ko9eq4julep2tfglnv4o18rz3py9dik4ywox0ytoscycn39o4vs",
+    "resourceId": "organization/c0bf7aac-7690-4905-a8aa-889df1510314/project/6e82a47b-79af-4920-ad42-c0f74421ab52/hashicorp.consul.global-network-manager.cluster/my-cluster"
+  },
+  "generation": "01HMHTHZND8VJDXHHJBKDR4TTA",
+  "id": {
+    "name": "global",
+    "tenancy": {
+      "peerName": "local"
+    },
+    "type": {
+      "group": "hcp",
+      "groupVersion": "v2",
+      "kind": "Link"
+    },
+    "uid": "01HMHTHZND8VJDXHHJBGY1KG0F"
+  },
+  "version": "60"
+}
+```
+
+### Link API vs. Configuration-based Linking
+
+The Link API described here is an alternative method to accomplish the same thing as [configuration-based linking](/consul/docs/agent/config/config-files#self-managed-hcp-parameters). You should generally only choose one method or the other for linking your cluster, not both. If you do use both methods, they interract in the following ways:
+
+* Values set in the configuration will take precedence over what was set in the API/CLI when Consul is started.
```

Comment:
```suggestion
* When Consul is started, values set in the `cloud` configuration will take precedence over what was previously set by the API or CLI.
```

Feel free to modify, but I wanted to move the "when consul is started" to the beginning of the sentence so that it's clear that the overwrite only happens at startup.